flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/SqlClient.java

```diff
@@ -184,8 +185,14 @@ private static void appendPythonConfig(Environment env, Configuration pythonConf
 
     public static void main(String[] args) {
         if (args.length < 1) {
-            CliOptionsParser.printHelpClient();
-            return;
+            args = new String[]{MODE_EMBEDDED};
```

Comment:
I think updating the `args` is not a good idea. Actually, we don't need to update args. You can use a `String mode` local variable and put remaining args into `final String[] modeArgs`.